### PR TITLE
HPCC-22453 Document Import ZAP Feature

### PR DESCRIPTION
**Pull request by g-pan**
**NOTE**: Used XXE 10.7 on unmanaged Linux workstation  

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

SUCCESSFUL INTEGRATION TESTING: https://github.com/g-pan/github-action-dev-build/actions/runs/9312847884

### CODE REVIEW

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Needs info about the importzap action.
`importzap <zapreport-filename> <output-helper-directory> [<zapreport-password>] `
needs to be added to the usage table on page 133.
Also, the reason why you may want to import one is in the Jira comments

File: docs/EN_US/HPCCSystemAdmin/SA-Mods/WUTool.xml [hunk 1]
```diff
@@ -82,6 +82,23 @@
...
         </row>
 
+        <row>
+          <entry>importzap</entry>
+
+          <entry>Imports ZAP report to be able to recreate a workunit and
+          replicate the reported issue. importzap requires the following
+          parameters. <para>&lt;zapreport-filename&gt;
+          &lt;output-helper-directory&gt;
+          [&lt;zapreport-password&gt;]</para></entry>
```
Comment (JamesDeFabia) [inline docs/EN_US/HPCCSystemAdmin/SA-Mods/WUTool.xml:92]: zapreport-password is optional. Also, it is unclear what **output-helper-directory** is.

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
See comments inline

**Review by JamesDeFabia — APPROVED** [review]
Looks good from my POV, but needs SME review before merging.

File: docs/EN_US/HPCCSystemAdmin/SA-Mods/WUTool.xml [hunk 2]
```diff
@@ -82,6 +82,25 @@
...
         </row>
 
+        <row>
+          <entry>importzap</entry>
+
+          <entry>Imports ZAP report to be able to recreate a workunit and
+          replicate the reported issue. <para>Importzap requires the following
+          parameters.
+          </para><para>&lt;zapreport-filename&gt;</para><para>&lt;output-helper-directory&gt;
+          [optional] temporary directory to unpack the zap report
```
Comment (jakesmith) [inline docs/EN_US/HPCCSystemAdmin/SA-Mods/WUTool.xml:92]: output-helper-directory is not optional.

**Review by jakesmith — COMMENTED** [review]
@g-pan 1 comment.

output-helper-directory ideally would be optional, but it is not.

**Review by jakesmith — APPROVED** [review]
@g-pan - looks good.

**Pull request merged**